libnavigation-base/src/main/java/com/mapbox/navigation/base/options/RerouteOptions.kt

```diff
@@ -0,0 +1,19 @@
+package com.mapbox.navigation.base.options
+
+import androidx.annotation.IntRange
+
+class RerouteOptions private constructor(
+    val avoidManeuverRadius: Int
```

Comment:
NN team is going to provide "seconds before maneuver" instead of "meters" for Alternatives. To align that we have to do the same